scripts/crowdin/pretranslate.mjs

```diff
@@ -0,0 +1,159 @@
+const projectId = process.env.CROWDIN_PROJECT_ID;
+const token =  process.env.CROWDIN_PERSONAL_TOKEN;
+
+const BASE_URL = `https://api.crowdin.com/api/v2/projects/${projectId}`;
+const preTranslateURL = `${BASE_URL}/pre-translations`;
+const REQUEST_INTERVAL_TIMEOUT = 10_000;
+const MAXIMUM_REQUEST_RETRIES = 12;
```

Comment:
Naming is a bit misleading. I suggest something like `MAXIMUM_PRETRANSLATATION_STATUS_CHECK` or `MAXIMUM_STATUS_CHECK` or `MAX_STATUS_CHECK`

Reply:
Fixed